content/60.methods.md

```diff
@@ -12,26 +12,36 @@ Below, we will describe the processes used to synthesize the literature.
-Manubot also provides a functionality to create a bibliography using digital object identifiers (DOIs), website URLs, or other identifiers such as PubMed identifiers and arXiv IDs [@doi:10.1371/journal.pcbi.1007128].
-Due to the needs of this project, project contributors also submitted pull requests to add support for clinical trial identifiers.
-<!-- Additional info-->
-Finally, the use of Manubot and GitHub allowed for scripted updates to be run each time the manuscript was generated.
-These updates were used to check that the manuscript was complete and to dynamically update information in the manuscript.
-<!-- Additional info about CI-->
+Manubot also provides a functionality to create a bibliography using digital object identifiers (DOIs), website URLs, or other identifiers such as PubMed identifiers and arXiv IDs.
+Due to the needs of this project, project contributors also implemented new features in Manubot and [Zotero](https://www.zotero.org/), which Manubot uses to extract metadata for some types of citations.
+These features support directly citing clinical trial identifiers such as `clinicaltrials:NCT04292899` [@clinicaltrials:NCT04292899].
+Finally, Manubot and GitHub Actions continuous integration allowed for scripted updates to be run each time the manuscript was generated.
+These scripts were used to check that the manuscript was built correctly, run spellchecking, and cross-reference the manuscripts cited in this review, summarized in Appendix A, and discussed in the project's issues and pull requests.
 
 #### Data Analysis and Visualization
 
-The combination of Manubot and GitHub Actions made it possible to integrate up-to-date analyses and visualizations of online data sources into the manuscript.
-Data about worldwide cases and deaths from the COVID-19 Data Repository by the Center for Systems Science and Engineering at Johns Hopkins University [@https://github.com/CSSEGISandData/COVID-19/tree/master/csse_covid_19_data/csse_covid_19_time_series] read using a Python script to generate Figure @fig:csse-deaths.
-Similarly, Figure @fig:ebm-trials was generated based on data from the University of Oxford Evidence-Based Medicine (EBM) Data Lab's COVID-19 TrialsTracker [@doi:10.5281/zenodo.3732709].
-In both cases, frequency data was plotted using MatPlotLib in Python.
-Figure @fig:ebm-map was generated using the countries associated with the trials listed in the EBM's COVID-19 TrialsTracker, converting the country names to 3-letter ISO codes using pycountry or manual adjustment when necessary, and visualizing the geographic distribution of trial recruitment using geopandas. 
-Current version information for packages and software is available in https://github.com/greenelab/covid19-review/blob/external-resources/environment.yml.
-
-<!-- Add in RNA-Seq analysis-->
+The combination of Manubot and GitHub Actions also made it possible to dynamically update information such as statistics and visualizations in the manuscript.
```

Comment:
I know you say statistics twice, but it's so unusual that I think folks might read right by it. What do you think about including one of the statistics here and inputting the variable?